### ISSUE TITLE
instance for `Rel8able (Generically1 t)` 

### ISSUE THREAD

**Issue by MangoIV** [circuithub/rel8#271]
Hi, i would be cool to additionally to the `anyclass` deriving mechanism provide an instance for 
- `Generically1` as defined in base `>=4.17`
- `Generically1` as defined here https://hackage.haskell.org/package/generically-0.1.1/docs/GHC-Generics-Generically.html for older base versions 

That would make it possible to 
`deriving Rel8able via (Generically1 MyType)` 

Thanks in advance.

**Comment by TeofilC** [comment]
I don't think this works because `Generically1 :: (Type -> Type) -> Type` but we want `(Context -> Type) -> Type`! Really `Generically1` should be kind polymorphic

**Comment by MangoIV** [comment]
I don't quite understand. `Generically1` seems to be kind polymorphic since at least base 4.19

**Comment by TeofilC** [comment]
Ah indeed that's right. I was misreading the error message that I saw.

I gave this another try and I was hoping that it would just be a case of adding some `coerce` calls, but it looks like that makes the typechecker quite unhappy.

I don't think I can assign more time to this unfortunately, but a PR would be welcome!

**Comment by MangoIV** [comment]
Thank you anyway! I don't personally use rel8 anymore so it's unlikely I'm going to report a fix myself. If you're okay with it, I would leave the issue open anyway?
